mkdocs.yml

```diff
@@ -49,6 +49,9 @@ nav:
     - Anti-phishing: getting-started/anti-phishing.md
     - Other email alias service: getting-started/compatibility-email-alias.md
     - Troubleshooting: getting-started/troubleshooting.md
+  - Two factor authentication:
```

Comment:
I think we need to remove the `/` before `two-factor-auth`

It'd be better to move this new section between Subscription and SIWSL so more frequently questions appear on top.

![CleanShot 2024-03-04 at 15 55 41](https://github.com/simple-login/documentation/assets/3167780/d96a3615-1a22-4360-8a49-ba9359a2aa43)